recipes/stata_kernel/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "stata_kernel" %}
+{% set version = "1.12.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stata_kernel-{{ version }}.tar.gz
+  sha256: 89885bd85f3debb62ee993e58f8108cb931a22d95885327cb53958914ed0bc7b
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools >=38.6.0
+    - twine >=1.11.0
```

Comment:
While twine is listed in `setup_requires` this is a really weird requirement and doesn't actually seem to be used for setup (or in the entire code base). I think it's just listed as a convenience for those building packages. Can you remove?

Reply:
Discussed it with the package maintainer and twine will be removed from setup_requires. However I don't know when the next release is planned with the updated requirements, so should probably leave it there for the time being as the build without it.

Reply:
Ah, my mistake.

Reply:
No worries, thanks for pointing it out.